File: API/FunctionAppDoc2Data/Models/ReceiptDashboardDTO.cs
using System;

namespace FunctionAppDoc2Data.Models;
public class ReceiptDashboardDTO
{
    public Guid ReceiptId { get; set; }
    public string ReceiptNumber { get; set; }
    public DateTime? ReceiptDate { get; set; }
    public string SupplierName { get; set; }
    public string SupplierAddress { get; set; }
    public string SupplierPhone { get; set; }
    public string CustomerName { get; set; }
    public string CustomerPhone { get; set; }
    public string CustomerAddress { get; set; }
    public decimal SubTotal { get; set; }
    public decimal? Discount { get; set; }
    public decimal? OtherCharge { get; set; }
    public decimal TotalAmount { get; set; }

}

File: src/app/interfaces/receipt-dashboard-dto.ts
export interface ReceiptDashboardDTO {
    receiptId: string;
    receiptNumber: string;
    receiptDate: string;
    supplierName: string;
    supplierAddress: string;
    supplierPhone: string;
    customerName: string;
    customerPhone: string;
    customerAddress: string;
    subTotal: number;
    discount: number;
    otherCharge: number;
    totalAmount: number;
}
